apihub/subscription/models.py

```diff
@@ -28,8 +28,7 @@ class Application(Base):
     url = Column(String)
     description = Column(String)
 
-    all_subscriptions = relationship("Subscription", backref="app")
-
+    subscriptions = relationship("Subscription", backref="app")
     subscriptions_pricing = relationship("SubscriptionPricing", backref="app")
```

Comment:
we have plural for the subscriptions, so probably we will have `subscription_pricings`?

Reply:
it's subscriptions_pricing